Workload-Security/Integration/aws-python-amazon-inspector-integration/c1-inspector-findings-protection.yaml

```diff
@@ -0,0 +1,837 @@
+                result = list()
+                # print('instance', instance_id)
+                resp = inspector.list_findings(
+                    filterCriteria={
+                        "findingStatus": [{"comparison": "EQUALS", "value": "ACTIVE"}],
+                        "resourceId": [{"comparison": "EQUALS", "value": instance_id}],
+                    }
+                )
+                result.extend(resp["findings"])
+                next_token = resp.get("nextToken")
+                while next_token:
+                    resp = inspector.list_findings(
+                        filterCriteria={
+                            "findingStatus": [{"comparison": "EQUALS", "value": "ACTIVE"}],
+                            "resourceId": [{"comparison": "EQUALS", "value": instance_id}],
+                        },
+                        nextToken=next_token,
+                    )
+                    result.extend(resp["findings"])
+                    next_token = resp.get("nextToken")
+
+                for finding in result:
+                    cve = finding.get("packageVulnerabilityDetails", {}).get("vulnerabilityId")
+                    if cve:
+                        cves.append(cve)
+                    # print(cve)
+                return cves
+
+
+            def all_instances_details_from_inspector(inspector):
+                instances = list()
+                result = list()
+                findings = inspector.list_finding_aggregations(
+                    aggregationType="AWS_EC2_INSTANCE",
+                    # aggregationRequest='AWS_EC2_INSTANCE',
+                )
+                result.extend(findings["responses"])
+                next_token = findings.get('nextToken')
+                while next_token:
+                    findings = inspector.list_finding_aggregations(
+                        aggregationType="AWS_EC2_INSTANCE",
+                        # aggregationRequest='AWS_EC2_INSTANCE',
+                        nextToken=next_token,
+                    )
+                    result.extend(findings["responses"])
+                    next_token = findings.get('nextToken')
+
+                for finding in result:
+                    instance_details = {
+                        "id": finding["ec2InstanceAggregation"]["instanceId"],
+                        "name": finding["ec2InstanceAggregation"].get("instanceTags", {}).get("Name"),
+                    }
+                    instances.append(instance_details)
+                print("Found total instances =", len(instances))
+                return instances
+
+
+            def find_rule_for_cve(all_rules, cve):
+                return [rule for rule in all_rules if cve in (rule.to_dict()["cve"] or [])]
+
+
+            def send_email(sender, recipients, subject, body, html_body):
+                print("sending email confirmation...")
+                # The character encoding for the email.
+                CHARSET = "UTF-8"
```

Comment:
Just to keep the variable name consistent, all the variables are the following:

**python_var**

Why is using a different [convention](https://www.w3schools.com/python/gloss_python_variable_names.asp) for this one?